CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # api
 
+## 8x.9.8 - 01 September 2022
```

Comment:
errr... somehow this file ended up in here after rebasing oO

Reply:
yeah, I'm interested how you have this commit locally but it's missing in main. I guess we'll just be naughty and merge it here anyway